.github/workflows/release.yml

```diff
@@ -0,0 +1,128 @@
+---
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-type:
+        description: How major the changes are [ major | minor | patch | prerelease ]
```

Comment:
So will we have to specify in our commit, answer a issue or it will just work?

Reply:
you can run it from the actions tab manually like this
![image](https://user-images.githubusercontent.com/20371518/171044116-614888b4-57ef-4243-99c6-2f29e3e26514.png)

Reply:
Ok cool!